issues/plan-unix-style-migration.md

```
@@ -0,0 +1,565 @@
+- `test_stream_emits_step_start`
+- `test_stream_emits_step_complete`
+- `test_stream_final_result_to_stdout`
+- `test_stream_progress_to_stderr`
+- `test_stream_disabled_by_default`
+- `test_stream_events_are_valid_jsonl`
+
+---
+
+## Implementation Order & Dependencies
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)